functions.sh

```diff
@@ -1,30 +1,48 @@
-startConjur() {
-  # Start a local Conjur to test against
-  docker-compose pull postgres conjur cuke-master
-  docker-compose up -d conjur cuke-master
+#!/bin/bash
```

Comment:
It doesn't make sense to have a shebang here -- this file is only ever sourced, not run.